README.md

```diff
@@ -11,45 +11,54 @@
     <img src="src/Assets/logo-readme.png" alt="Logo" width="80" height="80">
   </a>
 
-  <h3 align="center">Unofficial Netflix - README</h3>
+  <h3 align="center">Instagram Clone - README</h3>
 </p>
 
 
 
 <!-- TABLE OF CONTENTS -->
 ## Table of Contents
 
-- [Insgtram Clone)](#insgtram-clone)
-    - [Built With](#built-with)
+- [Instagram Clone](#instagram-clone)
+    - [Link to the application](#linktotheapp)
+    - [Built Using](#built-using)
   - [Getting Started](#getting-started)
     - [Prerequisites](#prerequisites)
     - [Installation](#installation)
-  - [Usage](#usage)
+  - [Usage Experience](#usage-experience)
   - [Road map](#road-map)
-  - [Contributing](#contributing)
+  - [Contribution](#contribution)
   - [Contact](#contact)
   - [Acknowledgements](#acknowledgements)
 
 
 
+
 <!-- ABOUT THE PROJECT -->
 ## About The Project
 <p align="center">
   <a href="https://github.com/abhishekpatel946/instagram-clone">
-<!--     <img src="Icon/logo-passmanager.jpeg" alt="Logo" width="100" height="100"> -->
+<!--     <img src="Icon/logo-passmanager.jpeg" alt="Logo" width="10" height="10"> -->
```

Comment:
remove this line of code which already comments.

Reply:
ok